{"name": "Проверка возвращаемого кода 404 при логине несуществующим курьером", "status": "passed", "start": 1723291323672, "stop": 1723291324056, "uuid": "b69de7f4-c5db-4828-875b-e1377c8f1471", "historyId": "fa25963727f53b3f362cf20a8027a5d7", "testCaseId": "fa25963727f53b3f362cf20a8027a5d7", "fullName": "tests.test_authorization_courier.TestAuthCourier#test_login_nonexistent_courier", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_authorization_courier"}, {"name": "subSuite", "value": "TestAuthCourier"}, {"name": "host", "value": "MacBook-Pro-Petr.local"}, {"name": "thread", "value": "19963-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_authorization_courier"}]}